src/com/layout/grid/grid.js

```diff
@@ -0,0 +1,22 @@
+import {h, Component} from 'preact/preact';
+import Col from 'com/layout/col/col';
+import Row from 'com/layout/row/row';
+import Container from 'com/layout/container/container';
+
+export default class Grid extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render( {children, columns = 3} ) {
+    return (
+      <Container {...this.props} class={cN([this.props.class, "-grid"])}>
+        {children.map((child, index) => {
+          return (
+            <Col flexGrow={0} flexBasis={100 / (columns)}>{child}</Col>
```

Comment:
`100 / columns`

Reply:
This is a remnant from when it had to be `100 / (columns - 1)` because flexbox can be annoying. Will fix.

Reply:
Makes sense. I'm assuming the code is more-or-less working properly now, so just catching minor things like this as I go.